calibration_with_blender/runner.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+import pathlib
+import shutil
+import subprocess
+import yaml
+import random
+
+
+def clear_dir(folder):
+    for filename in os.listdir(folder):
+        file_path = os.path.join(folder, filename)
+        try:
+            if os.path.isfile(file_path) or os.path.islink(file_path):
+                os.unlink(file_path)
+            elif os.path.isdir(file_path):
+                shutil.rmtree(file_path)
+        except Exception as e:
+            print('Failed to delete %s. Reason: %s' % (file_path, e))
+
+
+if __name__ == '__main__':
+    binary_path = '/home/xperience/development/opencv-fork/cmake-build-release/bin'
+    image_distort_path = os.path.join(binary_path, 'example_cpp_image_distort')
+    calibration_benchmark_path = os.path.join(binary_path, 'example_cpp_calibration_benchmark')
```

Comment:
What is difference between the code in PR and example in OpenCV. I propose to not add copy, but add reference to OpenCV itself. Also interactive calibration tool is always build with OpenCV and provided in distribution. It could be a replacement.